strings/arcade.json

```diff
@@ -114,14 +149,28 @@
 		"Lizard Man",
 		"Mecha-Gondola",
 		"Peteoid",
+		"Powergamer",
 		"Slime",
 		"Syndie",
+		"Tider",
 		"Uhangoid",
 		"Unicorn",
 		"Tunnel Clown",
 		"Vhakoid"
 	],
 
+	"rpg_enemy_valentines": [
+		"Aphrodite",
+		"Comdom",
+		"Ex-Boyfriend",
+		"Ex-Girlfriend",
+		"Maneater",
+		"Spess Casanova",
+		"Spess Don Juan",
+		"St. Valentine",
+		"Waifu"
```

Comment:
Waifu but no husbando?

Reply:
will fix this blatant sexism when I get home